formatter.go

```diff
@@ -565,15 +565,44 @@ func (f *DefaultFormatter) formatValue(value interface{}) string {
 				return ss
 			}
 		}
-		if b, err := json.MarshalIndent(value, "", defaultIndent); err == nil {
-			return string(b)
+
+		if arr, ok := value.([]interface{}); ok {
+			for k, v := range arr {
+				arr[k] = f.formatValue(v)
```

Comment:
Hi @gavv this is a WIP PR, I want to discuss first before continue.
So, I was thinking to handle nested slice / maps of floats by doing recursive call.
Example: `[[1.0, 2.0, 3.0]]` or `["a": [1.0, 2.0], "b": {"b1": [3.0, 4.0]}]`

WDYT?

Reply:
Makes sense